code/modules/modular_computers/file_system/programs/techweb.dm

```diff
@@ -208,7 +208,9 @@
 					var/obj/item/card/id/id_card_of_human_user = worn.GetID()
 					if(istype(id_card_of_human_user))
 						logname = "User: [id_card_of_human_user.registered_name]"
-			stored_research.research_logs = list(list(tech_node.display_name, price["General Research"], logname, "[get_area(src)] ([computer.x],[computer.y],[computer.z])"))
+			var/i = stored_research.research_logs.len
+			stored_research.research_logs += null
+			stored_research.research_logs[++i] = list(tech_node.display_name, price["General Research"], logname, "[get_area(computer)] ([user.x],[user.y],[user.z])")
```

Comment:
If you want to add list to a list without expanding it, just use `list += list(real_list)`

Reply:
Im stupid, thank you, fixed now. tested locally.